### DISCUSSION TITLE
Application set by TestFlight crushes when I deploy it from codemagic, but it doesn't when I publish by myself

### DISCUSSION THREAD

**Discussion by insignity** [codemagic-ci-cd/codemagic-docs#2419] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I set git listener, when I push my changes codemagic detects it and start creating ipa to send and when it delivers it in testflight it fails to open in Iphone X, Iphone 14 Pro Max too

### Expected results

I works fine from TestFlight

### Actual results

App crushes when I try to open

**Comment by himesh-cm** [comment]
Hi @insignity, This could be due to multiple reasons.
Can you please share the build ID with me? You can find it under your build details page's 'ID' section.  ﻿Navigate to the 'Builds' section on the left-hand menu, select the build you want to view, and click the copy icon.

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
